pkg/plugins/platform/local.go

```diff
@@ -812,13 +816,26 @@ func (l *LocalApp) Config() error {
 func (l *LocalApp) Down(removeData bool) error {
 	l.DockerEnv()
 	settingsFilePath := l.AppConfig.SiteSettingsPath
+	_, missingConfig := CheckForConf(l.AppRoot())
 
-	err := dockerutil.ComposeCmd(l.ComposeFiles(), "down", "-v")
-	if err != nil {
-		util.Warning("Could not stop site with docker-compose. Attempting manual cleanup.")
-		err = Cleanup(l)
+	if !fileutil.FileExists(l.AppRoot()) {
+		err := Cleanup(l)
```

Comment:
Hmm, I thought we were going to unconditionally do Cleanup() and skip the extra if/then/else stuff here, and simplify the code. Let's do that please, unless there was some reason not to do that.